web3swift/src/Account/EthereumKeyStorage.swift

```diff
@@ -9,28 +9,66 @@
 import Foundation
 
 public protocol EthereumKeyStorageProtocol {
-    func storePrivateKey(key: Data) throws -> Void
-    func loadPrivateKey() throws -> Data
+    func fetchStoredAddresses() throws -> [String]
+    func storePrivateKey(key: Data, with address: String) throws -> Void
+    func deletePrivateKey(for address: String) throws -> Void
+    func loadPrivateKey(for address: String) throws -> Data
```

Comment:
`address` should be `EthereumAddress`, and implementation in library should throw if `address` doesn't exist